### PR TITLE
{data}[GCCcore/9.3.0] MDSplus v7.96.8

### PR DESCRIPTION
**Pull request by Louwrensth**

### CODE REVIEW

**Comment by verdurin** [comment]
@Louwrensth please add the missing `sanity_check_paths` section.

File: easybuild/easyconfigs/m/MDSplus/MDSplus-7.96.8-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+preconfigopts = 'export CFLAGS="$CFLAGS -I$EBROOTLIBXML2/include/libxml2 " && '
+
+parallel = 1
+
+modextravars = {
+    'MDSPLUS_DIR': '%(installdir)s',
+    'MDS_PATH': '%(installdir)s/tdi',
+}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MDSplus/MDSplus-7.96.8-GCCcore-9.3.0.eb:38]: This is where the `sanity_check_paths` should go.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MDSplus/MDSplus-7.96.8-GCCcore-9.3.0.eb (hunk 1 above)
Comment (Louwrensth) [reply easybuild/easyconfigs/m/MDSplus/MDSplus-7.96.8-GCCcore-9.3.0.eb:38]: Thanks for the hint, added it.

**Comment by Louwrensth** [comment]
I would like to invite @SimonPinches to review this too.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3410dafb4f5bfe2e6d69f5e0e978ab66 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node106.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/66bfcbc1da6356cf182e18590152cca8 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @Louwrensth !

**Comment by SimonPinches** [comment]
Just to note, I put a newer version in the PR under review here: https://github.com/easybuilders/easybuild-easyconfigs/pull/10705

**Pull request merged**